crypto/serializer/deserializer_pkey.c

```diff
@@ -279,6 +280,28 @@ static void collect_name(const char *name, void *arg)
     data->error_occured = 0;         /* All is good now */
 }
 
+static void collect_deserializer(OSSL_DESERIALIZER *deser, void *arg)
+{
+    struct collected_data_st *data = arg;
```

Comment:
This is a bit easier to understand..

Reply:
Cool, I'm glad that makes a difference :-)